CMakeLists.txt

```diff
@@ -884,6 +883,23 @@ player_find_package(NAME lhasa
 	TARGET LHASA::liblhasa
 )
 
+# json support
+option(PLAYER_WITH_NLOHMANN_JSON "Support processing of JSON files" ON)
```

Comment:
Maybe move the condition to the option instead of package finding? Is not intuitive when the option has no effect.
-> something with set cache force for emscripten